docs/DeveloperGuide.md

```diff
@@ -2,7 +2,14 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Class Diagram2.svg)
+
+### Sequence digram for v2.0
+### Sequence digram for listAllLocations
+![Sequence Digram of listAllLocations](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Sequence%20Digram.JPG)
```

Comment:
Good morning, unfortunately it seems like the links to all of the diagrams aren't functional - maybe can try a relative path import instead?
![image](https://user-images.githubusercontent.com/57757704/113081168-c3f6ed00-920a-11eb-8d09-058ea0d35ecd.png)